modules/api/pkg/ee/clusterbackup/storage-location/handler.go

```diff
@@ -0,0 +1,282 @@
+//go:build ee
+
+/*
+                  Kubermatic Enterprise Read-Only License
+                         Version 1.0 ("KERO-1.0”)
+                     Copyright © 2024 Kubermatic GmbH
+
+   1.	You may only view, read and display for studying purposes the source
+      code of the software licensed under this license, and, to the extent
+      explicitly provided under this license, the binary code.
+   2.	Any use of the software which exceeds the foregoing right, including,
+      without limitation, its execution, compilation, copying, modification
+      and distribution, is expressly prohibited.
+   3.	THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND,
+      EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+      MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+      IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+      CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+      TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+      SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+   END OF TERMS AND CONDITIONS
+*/
+
+package storagelocation
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	velerov1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+
+	apiv2 "k8c.io/dashboard/v2/pkg/api/v2"
+	"k8c.io/dashboard/v2/pkg/handler/v1/common"
+	"k8c.io/dashboard/v2/pkg/provider"
+	kubermaticv1 "k8c.io/kubermatic/v2/pkg/apis/kubermatic/v1"
+	utilerrors "k8c.io/kubermatic/v2/pkg/util/errors"
+)
+
+type listCbslLReq struct {
+	common.ProjectReq
+}
+type getCbslReq struct {
+	common.ProjectReq
+	// in: path
+	// required: true
+	ClusterBackupStorageLocationName string `json:"clusterBackupStorageLocation"`
+}
+
+type createCbslReq struct {
+	common.ProjectReq
+	// in: path
```

Comment:
If I understand the Go tags correctly, then I think this should be an `// in: body` tag. At least this is what we have done in other places e.g. https://github.com/kubermatic/dashboard/blob/03648c694cca330f62ad519cac4338a2bd9af6f5/modules/api/pkg/handler/v2/application_installation/request.go#L41-L42